src/app/components/Sandbox.js

```diff
@@ -47,6 +48,9 @@ const SandboxComponent = ({ admin }) => {
   const [switchesDisabled, setSwitchesDisabled] = React.useState(Boolean(false));
   const [switchesHelp, setSwitchesHelp] = React.useState(Boolean(false));
   const [switched, setSwitchExample] = React.useState(Boolean(false));
+  const [limitedText, setLimitedText] = React.useState('Hello this is the initial limited text state');
+  // eslint-disable-next-line
+  console.log('~~~RENDER');
```

Comment:
Question: Is this intended because this is a sandbox?

Reply:
oops, will remove